tests/edx/test_mixins.py

```diff
@@ -0,0 +1,74 @@
+"""Tests of the mixins class."""
+
+from datetime import timedelta
+
+from faker import Faker
+
+from mork.edx.factories import EdxUserFactory
+from mork.edx.models import User
+
+
+def test_edx_usermixin_get_inactives(db):
+    """Test the `get_inactives` method."""
+    # 3 users that did not log in for 3 years
+    inactive_users = EdxUserFactory.create_batch(
+        3, last_login=Faker().date_time_between(end_date="-3y")
+    )
+    # 4 users that logged in recently
+    EdxUserFactory.create_batch(
+        4, last_login=Faker().date_time_between(start_date="-3y")
+    )
+
+    # Get all users inactive for more than 3 years
+    users = User.get_inactives(
+        db.session, inactivity_period=timedelta(days=365 * 3), offset=0, limit=9
```

Comment:
Why do you set limit to 9 ?

Reply:
Good point, changing them to `0`

Reply:
Reverting to `9`, as for SQLAlchemy and for SQL, a limit of `0` does not mean unlimited, but means returning an empty set

Reply:
And so, why 9 ? :sweat_smile: